frontend/components/arts/ArtistCard.vue

```diff
@@ -120,6 +137,9 @@ export default {
       return `By ${listOfArtist}`
     },
     returnMediaType() {
+      // This is a bit weird. You're not returning a mediaType, so this shouldn't be
+      // named as returnMediaType or simply mediaType. You're returning a PNG, so this
+      // should be something else. `mediaTypeImage`?
       let mediaType = ''
```

Comment:
This function seems to do nothing. Cant  the callee just call doneReveal directly?

Reply:
doneReveal is from a different function. Github awkwardly sliced this part. Here's the rest of the current function.
![image](https://user-images.githubusercontent.com/55416397/79895596-b7ceb100-8439-11ea-993d-327e0fe931a9.png)